utils/gitlab_api.py

```diff
@@ -188,6 +188,46 @@ def create_app_interface_reporter_mr(self, reports, email_schema,
         return self.create_mr(branch_name, target_branch,
                               commit_message, labels=labels)
 
+    def create_app_interface_notificator_mr(
+        self, notification,
+        notification_path="/app-interface",
+        email_schema="/app-interface/app-interface-email-1.yml"
+    ):
+        # labels = ['automerge']
+        labels = ['do-not-merge/hold']
+        prefix = 'app-interface-notificator'
+        target_branch = 'master'
+
+        now = datetime.now()
+        ts = now.strftime("%Y%m%d%H%M%S")
+        isodate = now.isoformat()
+        short_date = now.strftime('%Y-%m-%d')
+
+        branch_name = 'app-interface-notificator-{}'.format(ts)
+        commit_message = '[{}] notification for {}'.format(
+            prefix, isodate
+        )
+
+        self.create_branch(branch_name, target_branch)
+
+        msg = 'add email notification'
+        email_path = f"data{notification_path}/emails/{branch_name}.yml"
+        email = {
+            '$schema': email_schema,
+            'labels': {},
+            'name': branch_name,
+            'subject': f"[{prefix} {notification['notification_type']}]" +
```

Comment:
The plus sign is not needed here.
Also, it seems like there's a missing white-space after the closing square bracket.
Please align the string continuation line to the beginning of the string in favor or readability.

Reply:
Gotcha!